src/WebdriverClassicDriver.php

```diff
@@ -924,12 +925,15 @@ private function charToSynOptions($char, ?string $modifier = null): string
      * Executes JS on a given element - pass in a js script string and argument[0] will
      * be replaced with a reference to the result of the $xpath query
      *
+     * Example:
+     * ```
+     * $this->executeJsOnXpath($xpath, 'return argument[0].childNodes.length');
+     * ```
+     *
      * @param string $xpath the xpath to search with
      * @param string $script the script to execute
-     *
      * @return mixed
      * @throws DriverException
-     * @example $this->executeJsOnXpath($xpath, 'return argument[0].childNodes.length');
```

Comment:
Note that according to PHPDoc standard, `@example` tags are supposed to point to source code files, and not be the example itself (which makes the tag utterly useless in most cases, but anyway).